src/metadata-stream.js

```diff
@@ -6,12 +6,24 @@
 (function(window, videojs, undefined) {
   'use strict';
   var
```

Comment:
Yep, part of the idea for forking standard to videojs-standard was that we could have it included in all our projects.